src/sentry/models/integrationfeature.py

```diff
@@ -29,34 +29,39 @@ def as_choices(cls):
 
```

Comment:
I cleaned up some of the unnecessary `elif` conditions and moved the API case to the end as a default catch-all.